### PR TITLE
Add pending_signing_tag

### PR DESCRIPTION
**Pull request by puiterwijk**
The idea is that instead of into pending_testing_tag, we tag into pending_signing_tag,
and then robosignatory will pick them up, sign them, and move them to pending_testing_tag.

Signed-off-by: Patrick Uiterwijk puiterwijk@redhat.com

### CODE REVIEW

**Comment by bowlofeggs** [comment]
Thanks @puiterwijk - this idea makes sense to me. Could you add some test coverage for the new lines? Also, it seems that we should have a data migration to add this new tag to existing repos.

**Comment by puiterwijk** [comment]
I was not intending to do migration inside bodhi. We can just run the standard sigulsign_unsigned for one last time when we enable this.
That means that the same "everything in pending_testing_tag is signed" statement holds both before and after enabling this, without bodhi having to care abut it either way.

File: bodhi/server/models/models.py [hunk 1]
```diff
@@ -1129,7 +1133,7 @@ def set_request(self, db, action, username):
         # Add the appropriate 'pending' koji tag to this update, so tools like
         # AutoQA can mash repositories of them for testing.
         if action is UpdateRequest.testing:
-            self.add_tag(self.release.pending_testing_tag)
+            self.add_tag(self.release.pending_signing_tag)
```
Comment (bowlofeggs) [inline bodhi/server/models/models.py:1136]: What process will add the pending_testing_tag when it is time for that?

**Review by bowlofeggs — APPROVED** [review]

File: bodhi/server/models/models.py (hunk 1 above)
Comment (puiterwijk) [reply bodhi/server/models/models.py:1136]: That will be robosignatory's duty.

**Pull request merged**